vllm/config.py

```diff
@@ -2149,6 +2150,11 @@ class CompilationConfig(BaseModel):
     level: int = 0
     backend: str = ""
     custom_ops: List[str] = Field(default_factory=list)
+    splitting_ops: List[str] = Field(default_factory=lambda: [
+        "vllm.unified_flash_attention",
+        "vllm.unified_flash_infer",
+        "vllm.unified_v1_flash_attention",
+    ])
```

Comment:
Should xformers be added?

Reply:
xformers are not supported for `torch.compile` right now.